Tests/SwiftFormatPrettyPrintTests/IfStmtTests.swift

```diff
@@ -283,4 +283,26 @@ public class IfStmtTests: PrettyPrintTestCase {
 
     assertPrettyPrintEqual(input: input, expected: expected, linelength: 50)
   }
+
+  public func testConditionExpressionOperatorGrouping() {
+    let input =
+      """
+      if someObj is SuperVerboselyNamedType || someObj is AnotherPrettyLongType  || someObjc == "APlainString" || someObj == 4 {
```

Comment:
Can you also add an example where one of the interior expressions (between two `||`s) is so long that it needs to wrap inside? Since you used `.continue` breaks, my hunch is that they won't stack, and you'll end up with something like this:

```swift
if something is SomeType
  || someOtherThing + someDifferentValue
  / someDenominator == 72
  || someLastCondition
{
```

which makes me think we might want to use `.open/.close` breaks around our `&&/||` groups instead of `.continue` breaks. Does that end up working correctly?

Reply:
It ended up being a little more involved, but I've got the intra-expression continuation indent working.